### ISSUE TITLE
Unminified code

### ISSUE THREAD

**Issue by mkalish** [codeisland/RI-Budget-Sankey#12]
I was inspired by this project and wanted to do a slightly modified version for Washington D.C.  Would it be possible to get access to the unminified code?  Also, was wondering how/if you wanted to be attributed.

**Comment by goldenmeanie** [comment]
@jamesrhaley did you see this question?

**Comment by jamesrhaley** [comment]
@mkalish I do apologize but I currently do not have the time to offer support on the code.  There are a few parts I did not open source on the graph, mainly the tooltip, and I do not have the time to rework the code to reflect that.  Sorry for the inconvenience.

**Comment by mkalish** [comment]
@jamesrhaley I'm mostly just looking for the unminified version of this file: new_sankey.js,  However, I understand if you don't have time.  Thanks for getting back to me.

**Comment by gregjd** [comment]
@mkalish - Perhaps you've tried this already, but I just ran the code through http://jsbeautifier.org/ and it does a pretty good job of remaking the unminified version, granted the variable names are lost.

I found this https://github.com/jamesrhaley/d3CodeIsland.github.io/blob/gh-pages/javascripts/main.js, but it looks like it might have been an earlier version (?). I haven't tried running it / inspecting each line. But perhaps this will help with the variable names.
